vNext/shared/AppInsightsCommon/src/Util.ts

```diff
@@ -709,11 +709,11 @@ export class DateTimeUtils {
      * Gets duration between two timestamps
      */
     public static GetDuration = function (start: number, end: number): number {
-        var result = null;
-        if (start !== 0 && end !== 0 && !CoreUtils.isNullOrUndefined(start) && !CoreUtils.isNullOrUndefined(end)) {
-            result = end - start;
+        var duration = undefined;
+        if (!(isNaN(start) || isNaN(end))) {
```

Comment:
no 0 checking is being done anymore

Reply:
0 is a valid start number right?, this was the reason I was not able to use this Util in pvp scenarios, Math.max will take care of negative numbers if end is 0 or less

Reply:
If it was blocking you, it's probably fine to remove. I think for our case 0 was never a valid start time